### ISSUE TITLE
[Question] Project Status

### ISSUE THREAD

**Issue by FaySmash** [winyl-player/winyl#19]
Is this Project dead or did this Repo move elsewhere?

**Comment by zag2me** [comment]
It would be a real shame as I found this software on reddit discussion about the "best" music player and I believe it actually is! This could be a real market leader if it was actively developed over time. I even would donate to the Patreon but it seems dead at the moment so I didnt bother.

**Comment by zag2me** [comment]
Sure already did that.. but no reply on the issue of forum registration. see https://github.com/winyl-player/winyl/issues/21 and https://github.com/winyl-player/winyl/issues/22

Hardly a marketer... just a grateful user to find a piece of software that I like very much.

**Comment by alexkx** [comment]
Something happened in my life, I'm not sure if I'll have time to add major features but I still continue to support the project and fix bugs.
Actually in the current state of the project there isn't any critical bugs or issues so you can continue to use it without any problems.

And I plan to release update with all latest changes and x64 version soon.

**Comment by FaySmash** [comment]
@alexkx guess we're all just relieved that you're doing well ^.^ and we're also looking forward for the next release, keep up the great work!
btw, it would be nice to have one universal installer for 32/64bit and local/portable installation :)

**Comment by Mlinotest** [comment]
Just wanted to say that Winyl is easily the best music player out there. I just can't believe how simple, intuitive, robust and good looking it is. It is everything I was looking in a music player ever since I first saw Winamp.
I just hope alexkx finds some time to update it with new features. Maybe we could help by donating?

**Comment by Doekoeman** [comment]
I agree about Winyl about being the best music player out there......

alexkx is gone AWOL  ->  Donating would be a good idea....!!    
I'm sure the Reddit community would donate heavily too, to see new features....
